### PR TITLE
Ft fix pep8 errors 150398504

### PR DESCRIPTION
**Pull request by brayoh**
#### What does this PR do?
Fix Python coding style warnings/errors

#### Description of Task to be completed?
many errors ensue from running flake8. these require fixing.

#### How should this be manually tested?
cd to the project directory and run `pep8 .`
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
150398504
#### Screenshots (if appropriate)
N/A
#### Questions:
N/A

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -15,17 +15,12 @@ sudo: false
...
   - "3.5"
 
 # Manually define here the combinations environment variables to test
 # https://github.com/travis-ci/travis-ci/issues/1519
 env:
   - TEST_MOBILE=True  DB=postgresql TRAVIS_NODE_VERSION="4"
-  - TEST_MOBILE=True  DB=sqlite     TRAVIS_NODE_VERSION="4"
-  - TEST_MOBILE=False DB=postgresql TRAVIS_NODE_VERSION="4"
-  - TEST_MOBILE=False DB=sqlite     TRAVIS_NODE_VERSION="4"
-
```
Comment (egichuri) [inline .travis.yml:28]: changes to travis are being handled in a different branch

**Review by egichuri — CHANGES_REQUESTED** [review]

**Comment by egichuri** [comment]
let's change the base branch for this pr to `develop`

**Comment by egichuri** [comment]
@brayoh please change the base branch for this

**Comment by brayoh** [comment]
@egichuri I've changed the branch to develop

**Review by egichuri — APPROVED** [review]

**Pull request merged**